typescript/infra/scripts/helloworld/utils.ts

```diff
@@ -50,9 +50,13 @@ export async function getApp<Chain extends ChainName>(
     helloworldConfig.addresses,
     helloWorldFactories,
   ) as ChainMap<Chain, HelloWorldContracts>;
-  const multiProvider = await coreConfig.getMultiProvider();
-  const app = new HelloWorldApp(contracts, multiProvider as any);
-  return app;
+  const multiProvider =
```

Comment:
is `const multiProvider: MultiProvider<any> = ...` sufficient here? I find it slightly more readable, but not passionate